public/assets/js/filtrecategorie.js

```diff
@@ -0,0 +1,34 @@
+window.onload = () => {
+    const FiltersForm = document.querySelector("#filtres");
```

Comment:
```suggestion
    const filtersForm = document.querySelector("#filtres");
```
lower case var names